examples/touch/content/TabBarPage.qml

```diff
@@ -74,23 +74,23 @@ Page {
             frame: Item { }
             tab: Item {
                 implicitWidth: control.width/control.count
-                implicitHeight: 50
+                implicitHeight: size.dp(50)
```

Comment:
It temprary silution. I will add in a short time theme class and put all calculation on it.